docs/uk/docs/index.md

```diff
@@ -0,0 +1,465 @@
+<p align="center">
+  <a href="https://fastapi.tiangolo.com"><img src="https://fastapi.tiangolo.com/img/logo-margin/logo-teal.png" alt="FastAPI"></a>
+</p>
+<p align="center">
+    <em>Фреймворк FastAPI, висока продуктивність, простий у вивченні, швидкий для написання коду, готовий до продакшину</em>
```

Comment:
```suggestion
    <em>Фреймворк FastAPI, висока продуктивність, простий у вивченні, швидкий для написання коду, готовий до продакшину</em>
```

Reply:
this does not loooks good

Reply:
@akarev0 As I understood these are keywords at the top of the page. Not sentences